dangerfile.ts

```diff
@@ -136,10 +136,6 @@ async function checkUiPackage() {
     fail('New composites were added, but JSXTypeUtil.ts is not modified')
   }
 
-  if (is_new_composites_added && !types_util_diff?.added.includes('../composites')) {
```

Comment:
Why removing this rule?

Reply:
we no longer export components from the index !